api/app/Console/Kernel.php

```diff
@@ -18,14 +20,30 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
+        // Log facade does not seem to work in here!
+        // Config utility does not seem to work in here!
+
         // $schedule->command(HardDeleteOldUsers::class)->dailyAt('08:00');
 
-        // clean up old user generated files every day at 2:00 AM
+        // clean up old user generated files every day at 1:00 AM
         $schedule->command(PruneUserGeneratedFiles::class)
             ->timezone('America/Toronto')
-            ->dailyAt('2:00')
+            ->dailyAt('1:00')
```

Comment:
Unrelated fix.  2:00 is a terrible time to schedule something because of DST changes.  It never happens on "spring forward" days and happens twice on "fall back" days.